.circleci/config.yml

```diff
@@ -381,12 +377,8 @@ jobs:
           name: Build XCFramework archive
           command: |
             bash megazords/ios-rust/build-xcframework.sh --build-profile release
-      - save_cache:
-          name: Save sccache cache
-          key: sccache-cache-macos-{{ arch }}-{{ epoch }}
-          paths:
-            - "~/Library/Caches/Mozilla.sccache"
-      - sccache-stats-and-logs
+      - save-sccache-cache:
+          path: "~/Library/Caches/Mozilla.sccache"
```

Comment:
I made the `save-sccache-cache` take a parameter to differentiate between the locations of the cache and have all executors use the same one